ChineseFilter/sections/specific.txt

```diff
@@ -16,6 +16,7 @@ kocpc.com.tw##.kocpc_bot_ad
 ||store.7box.vip/ad/
 ||syndication.exosrv.com/splash.php?$xmlhttprequest,redirect=nooptext,important,domain=hboav.com
 sssam.com##.widget_html_ad
+india.com##.home-widget-leadStory
```

Comment:
Not an ad

Reply:
Yes, the issue can be closed unless OP gives more info.

Reply:
> Not an ad

I agree with you.